testgrid/config/config.yaml

```diff
@@ -1611,6 +1620,17 @@ dashboards:
   - name: gci-1.4-gci-latest-upgrade-master
     test_group_name: kubernetes-e2e-gke-gci-1.4-gci-latest-upgrade-master
 
+- name: latest-upgrades
```

Comment:
there's a `google-gke-latest-upgrade` dashboard... what would be the difference? Assume those are 1.5-latest upgrade jobs?